phpcs.xml.dist

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<ruleset name="Beans">
+    <description>The code standard for Beans.</description>
+    <file>.</file>
+    <exclude-pattern>dist/*</exclude-pattern>
+    <exclude-pattern>node_modules/*</exclude-pattern>
+    <exclude-pattern>vendor/*</exclude-pattern>
+    <arg name="extensions" value="php"/>
+    <arg value="sp"/>
+    <arg name="report" value="full"/>
+    <arg name="report" value="summary"/>
+    <arg name="report" value="source"/>
+    <rule ref="PHPCompatibility" />
+    <config name="testVersion" value="5.2-"/>
+    <rule ref="WordPress">
```

Comment:
New empty line + comment 

```

    <!-- WordPress Rules -->
    <rule ref="WordPress">
        <exclude name="WordPress.VIP"/>
    </rule>
```

And @GaryJones, let's add a space between each of the `<rule...></rule>` blocks too, which I'll note below.